### PR TITLE
Python

### PR DESCRIPTION
**Pull request by haggi**
Updated python scripts and got rid of unneeded entries in the module file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 build/
 *.pyc
+.project
```
Comment (dictoon) [inline .gitignore]: You can actually ignore files in your repo only by modifying `.git/info/exclude` (it works just like `.gitignore`). I would recommend doing this for files specific to the tools you're using.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+.pydevproject
+*.oso
```
Comment (dictoon) [inline .gitignore]: Not only we should ignore `.oso` files, but we should also delete them from the repo.

**Comment by dictoon** [comment]
I'm slightly confused now: why did you remove the OSL files? I thought we only wanted to delete the *.oso files (and then add them to .gitignore).

**Comment by haggi** [comment]
I removed all the osl files which are not used by the plugin at all. They are completly useless at the moment because there is no way to use them from within the plugin. Only the ones in the maya subfolder are used. We can add them later if we need them.

**Comment by dictoon** [comment]
Thanks!

**Pull request merged**